src/components/CounterPage/CounterContainer/index.js

```diff
@@ -0,0 +1,64 @@
+import { Component } from "react";
+
+import Counter from "../Counter";
+
+class CounterContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      countValue: 0,
+      isEven: true
+    }
+  }
+
+  handleIncrease = () => {
+    this.setState(() => { return { countValue: this.state.countValue + 1 } });
```

Comment:
1) if you are using callback in setState, you have an access to state argument in callback, thats why you haven't call this.state.countValue
2) you can remove `return` in this case. Just add bracers.
ex `() => ({
   ...your code
})`